sonar-db/src/main/java/org/sonar/db/version/AddColumnsBuilder.java

```diff
@@ -79,6 +81,18 @@ private void addColumns(StringBuilder sql, String columnPrefix) {
 
   private void addColumn(StringBuilder sql, ColumnDef columnDef) {
     sql.append(columnDef.getName()).append(" ").append(columnDef.generateSqlType(dialect));
+    Object defaultValue = columnDef.getDefaultValue();
+    if (defaultValue != null) {
+      sql.append(" DEFAULT ");
+      // TODO remove duplication with CreateTableBuilder
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)